### DISCUSSION TITLE
How to use Glance

### DISCUSSION THREAD

**Discussion by lisamarcus7788** [community/community#148012] [Programming Help]
### Select Topic Area

Question

### Body

I've been using Glance for a while now and it's honestly pretty cool. But I feel like I’m not using it to its full potential. Like, what’s the best way to set it up or get the most out of it? If anyone has tips or knows how to make it better

**Comment by lizi83412** [comment]
To use Glance, simply visit the official site or download the app from a trusted source. At [here](https://teachingfeeling.pro/), you'll find guides, updates, and detailed instructions to ensure a smooth installation and optimal use. Be sure to review any available resources to help understand how to get the most out of its features. For more information and updates, you can check out the full site for instructions tailored to your device

**Comment by oliveirasmatheus** [comment]
Hi! It’s great to hear you’re enjoying Glance. I’ve used it for a while too, and I’ve picked up a few tips that might help you make the most out of it:

- Customize your dashboard: Take some time to tailor the widgets and metrics to what matters most for your workflow. It’s easy to get overwhelmed with information, so focusing on key insights can really enhance your experience.
- Integrate with other tools: Glance works well with several platforms, like GitHub Actions or project management tools. If you haven’t already, explore the integrations to streamline your workflow further.
- Set up notifications: Glance can send notifications for key events. Setting up smart notifications can keep you informed without having to constantly check.
- Explore community resources: The Glance community often shares innovative setups or configurations. You might find inspiration there to improve your setup.
- Experiment and iterate: The best configuration often comes from trial and error. Don’t hesitate to try different setups to see what works best for you.

Hope this helps! If you’ve already got a particular setup and want to share what works for you, I’d love to hear about it too.
